buddyboss/buddyboss-platform
# Privacy policy link the_content filter issue fixed

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #2424  .

### How to test the changes in this Pull Request:

1. Add the below snippet in child themes functions.php file.
add_filter( 'the_content', 'filter_the_content_in_the_main_loop', 1 );
function filter_the_content_in_the_main_loop( $content ) {
// Check if we're inside the main loop in a single Post.
if ( is_singular() && in_the_loop() && is_main_query() ) {
// uncomment the line below to make it work with BuddyBoss.
//remove_filter( 'the_privacy_policy_link', 'bp_core_change_privacy_policy_link_on_private_network', 999999 );
return $content . get_the_privacy_policy_link();
}
return $content;
}
2. See no page or post page will crash now.

### Other information:




### Changelog entry

> Privacy policy link the_content filter issue fixed
